### PR TITLE
fix logs

### PR DESCRIPTION
**Pull request by nnat425**

### CODE REVIEW

File: app/assets/stylesheets/vendor/jquery-ui.css [hunk 1]
```diff
@@ -799,15 +799,15 @@ body .ui-tooltip {
 }
 .ui-widget-content {
 	border: 1px solid #aaaaaa;
-	background: #ffffff url(images/ui-bg_flat_75_ffffff_40x100.png) 50% 50% repeat-x;
+	background: #ffffff 50% 50% repeat-x;
```
Comment (LucasCioffi) [inline app/assets/stylesheets/vendor/jquery-ui.css]: @nnat425 does this have any change for the user?

File: app/models/user.rb [hunk 2]
```diff
@@ -100,11 +113,10 @@ def validate_pin_pattern
   # a success/failure response.
   # Accepts a response from the microservice, and returns.
   def send_request_to_patron_creator_service
-    Rails.logger.debug("send_user_to_patron_creator_service: start")
     query = {
       'names' => [last_name.upcase + ', ' + first_name.upcase],
       'emails' => [email],
-      'pin' => pin,
+      'pin' => '1234',
```
Comment (LucasCioffi) [inline app/models/user.rb]: Is this hardcoded change in here accidentally?

Comment (nnat425) [reply app/models/user.rb]: Yes! Good catch.

File: app/assets/stylesheets/vendor/jquery-ui.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (nnat425) [reply app/assets/stylesheets/vendor/jquery-ui.css]: None at all. These images were not even stored in the repo from the start.

File: config/environments/local.rb [hunk 3]
```diff
@@ -22,6 +22,9 @@
   # Only use best-standards-support built into browsers
   config.action_dispatch.best_standards_support = :builtin
 
+  # Turn off asset pipline information showing in logs 
+  config.assets.logger = false
```
Comment (nnat425) [inline config/environments/local.rb]: @daryanypl @LucasCioffi As mentioned per slack, setting this to true will turn off requested assets from being printed out in the logs. What are both of your thoughts on this? I'd like to get some perspective.

File: app/assets/stylesheets/vendor/jquery-ui.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (LucasCioffi) [reply app/assets/stylesheets/vendor/jquery-ui.css]: @nnat425 Ok then we can remove `50% 50% repeat-x` which positions the background image.  So all that is left is setting the color to `#ffffff`.

File: config/environments/local.rb (hunk 3 above)
Comment (nnat425) [reply config/environments/local.rb]: Per conversation with Darya, we're going to leave it in for now.

**Pull request merged**